getting_started/bootstrap.rst

```diff
@@ -57,11 +57,31 @@ Start by creating a ``terraform.tfvars`` file. There is a template that you can
 - **cluster_prefix**: every resource in your tenancy will be named with this prefix
 - **KuberNow_image**: name of the image that you previously imported using Ansible
 - **ssh_key**: path to your public ssh-key to be used (for ssh node access)
-- **floating_ip_pool**: a floating IP pool name
+- **floating_ip_pool**: a floating IP pool label
 - **external_network_uuid**: the uuid of the external network in the OpenStack tenancy
 - **dns_nameservers**: (optional, only needed if you want to use other dns-servers than default 8.8.8.8 and 8.8.4.4)
 - **kubeadm_token**: a token that will be used by kubeadm, to bootstrap Kubernetes. You can run generate_kubetoken.sh to create a valid one.
 
```

Comment:
Sure thing, no problem in moving this and other info into the troubleshoot section. However, was wondering that usually in troubleshooting you should create extra content that is meant to help with errors that are not directly dependent with the basic and mandatory configuration settings that a generic user has to enter as input (e.g. the missing DNS python module scenario; user cannot foresee it, just has to debug it, hence a nice troubleshoot tip).

On the other hand it is true that non tech-users will mainly and actually run phenomenal from within the portal, thus the only people dealing with this repo are likely IT skilled and/or tech-enthusiastic. Nevertheless, we might want to allow some enthusiastic with a not in-depth knowledge of platform as OpenStack to be able to smoothly spin up the cluster and deploy a simple first app.

Anyway, this was just a long digression! What do you think? Let's move this new content anyway into the troubleshoot section?